code/modules/mob/living/carbon/alien/larva/powers.dm

```diff
@@ -9,9 +9,11 @@
 
 	if(layer != TURF_LAYER+0.2)
 		layer = TURF_LAYER+0.2
+		plane = TURF_LAYER
```

Comment:
Bug found here. Should be PLANE_TURF - this actually caused aliens when hiding to be visible beyond the darkness.